persist/sqlite/migrations.go

```diff
@@ -144,6 +145,12 @@ egress_limit, dyn_dns_provider, dns_update_v4, dns_update_v6, dyn_dns_opts, regi
 	return nil
 }
 
+// migrateVersion6 adds the sector_cache_size column to the host_settings table
+func migrateVersion7(tx txn) error {
```

Comment:
Move this to the top of the file and add the function to the `migrations` slice at the bottom.